### ISSUE TITLE
Layer Menu Persistence

### ISSUE THREAD

**Issue by RyanDuff613** [SketchIO/feedback-summer-2020#26]
![layerMenuPersistence](https://user-images.githubusercontent.com/48165321/91352040-1c11d080-e79e-11ea-9c2c-c0db6a768cc9.gif)
This issue is occurring after an extensive amount of time manipulating layers. The selected layers of the Layers menu are persisting over my image. If I hover over the "ghost layers", the cursor changes to a hand, as if the menu is still available but clicking on them has no effect. I'm able to open the Layer menu and interact with it as usual, though the "ghost menu" remains visible. Closing the layer menu does not remove the "ghost menu". Reloading the browser eliminates the issue.

**Comment by 3t8U** [comment]
I also came across this one today. 
<img width="963" alt="Screen Shot 2020-09-01 at 2 34 59 PM" src="https://user-images.githubusercontent.com/56529123/91908423-609ae180-ec60-11ea-9c5a-d55e4463305f.png">

**Comment by ryanreece** [comment]
@3t8U Thanks for the report! 

I literally just pushed a new version of Sketchpad so can you do a hard refresh and check to see if this happens again?

To do a hard refresh you can press and hold the shift key on your keyboard and then click the reload button in Chrome. Thanks!!

**Comment by 3t8U** [comment]
@ryanreece  - I'm on it !

**Comment by ryanreece** [comment]
@RyanDuff613 Can you also check on your end to see if this issue still happens? Thank you!

**Comment by RyanDuff613** [comment]
Will do!

**Comment by RyanDuff613** [comment]
So far so good with Sketchpad. I've been using it quite a bit and so far
haven't run into the persistence issue.

Well done!

**Comment by RyanDuff613** [comment]
Spoke too soon and ran into this one again just now. Another case where it didn't become an issue until I got a large number of layers. I had also just deleted 10+ stamp layers. Will try to recreate and see if I can provide more context.
<img width="1119" alt="Screen Shot 2020-09-02 at 10 30 33 AM" src="https://user-images.githubusercontent.com/48165321/92016722-9c3cb680-ed07-11ea-8268-653dd91db202.png">
<img width="1121" alt="Screen Shot 2020-09-02 at 10 30 46 AM" src="https://user-images.githubusercontent.com/48165321/92016750-a65eb500-ed07-11ea-97ee-eb3eaa8b0a27.png">
